src/store/store.js

```diff
@@ -62,14 +62,21 @@ const saveStateToLocalStorage = (state) => {
  * @param {Object} initialState Initial Redux state (from initial-state.js)
  * @return {Object} Redux store
  */
-export default function configureStore(initialState) {
+export default function configureStore(initialState, dataType) {
```

Comment:
Nit: perhaps add this new argument to the function's comment above?

Reply:
I've added the description for this parameter - please see my newer commits.